### ISSUE TITLE
Comps for Landing Page and Search Results page

### ISSUE THREAD

**Issue by cweber** [sunlightlabs/openfederal#21]
Comp out landing page and search results page based on wireframes and element board.

**Comment by cweber** [comment]
Alright...I'm movin' fast!

Comps are based on the element board that was shared and approved.

In the wireframes, I had saved space for an icon from each property, but some of them don't have that and it occurred to me that we may want to move away from highlight the unique branding of each of them, so I've adopted the text only reference that was already styled in the moodboards. If we want to include a small icon along with that text for properties that have one, I think i could make that work, but still maintain the subtlety.

**To note**

* The intent is for all images to be monotone, based on #3c4657...someone start thinking of an easy way to process images on the fly?
* The background image is of a Blue Sky.
*Also of note, my intent was actually do the trendy video background thing (only on the homepage), but we'll need to [purchase the video](http://www.istockphoto.com/video/time-lapse-us-capitol-dome-congress-and-cloudy-sky-14636124?st=d6c249d) and then modify it's color first, so for now all we get to see is the flat graphic. 

![landing-page](https://cloud.githubusercontent.com/assets/417657/10029434/e2cc7a62-613f-11e5-87bc-0df89042a7c1.png)


![search-results](https://cloud.githubusercontent.com/assets/417657/10029444/e7ff4e7e-613f-11e5-82a5-60cd2bdcfa92.png)


![search-results_filter](https://cloud.githubusercontent.com/assets/417657/10029446/ebabef64-613f-11e5-9a2c-b13c3ea3f014.png)

**Comment by amycesal** [comment]
Watch the contrast of the white text on yellow and especially the yellow text on gray. Check http://accessibility.oit.ncsu.edu/tools/color-contrast/ for measuring color contrast accessibility if you're interested.

**Comment by jalbertbowden** [comment]
nice. i care about a11y because i'm a web developer, but i care about color contrast because i'm color blind.
nice link. i use these two:
http://colorsafe.co/
http://leaverou.github.io/contrast-ratio/

**Comment by cweber** [comment]
Noted. I was worried about it and willfully ignored the problem. I do like the yellow as the action color, but I'll work something else out for buttons. It looks like the Dark navy blue from the brand colors on top of yellow is much better according to one of the tools I tried.

**Comment by heyitsolivia** [comment]
Yeah– @ngrad and I had talked about making the buttons black/dark text on yellow instead of white for increased contrast (in the element board) but we didn't get around to updating it.

**Comment by crdunwel** [comment]
I'm not colorblind but white on yellow is not very easy for me to discern. I like the idea of action buttons having yellow though to fit in with sunlight.

**Comment by crdunwel** [comment]
Also these look incredible. Wow.

**Comment by cweber** [comment]
And then I start moving slow. :)

Here are the search result comps with revised button styles and a mobile view. The yellow filter bar felt too light once the logo had been switched to just the icon. In general, I still have some messin' to do with the mobile view.


![search-results_small](https://cloud.githubusercontent.com/assets/417657/10312125/db75b7aa-6c17-11e5-9216-89709a3ce295.jpg)

![search-results_filter_medium](https://cloud.githubusercontent.com/assets/417657/10312124/db730186-6c17-11e5-9b19-11c6cd0b6bd0.jpg)

![search-results_filter](https://cloud.githubusercontent.com/assets/417657/10312126/db77a380-6c17-11e5-84ab-742da4f485d1.jpg)

**Comment by ngrad** [comment]
@cweber really cool to see these come to life.
